lopper/lops/lop-domain-a72-prune.dts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2020 Xilinx Inc. All rights reserved.
```

Comment:
Minor 'nit and no need to change this for this pull request, but this should be updated 2022 for new files, even if we are just moving things around.